catalyst_voices/apps/voices/lib/configs/bootstrap.dart

```diff
@@ -85,6 +86,18 @@ GoRouter buildAppRouter({
   );
 }
 
+@visibleForTesting
+Future<void> restartDependencies() async {
+  await Dependencies.instance.reset;
+}
+
+@visibleForTesting
+Future<void> registerDependencies() async {
+  if (!Dependencies.instance.isInitialized) {
+    await Dependencies.instance.init();
+  }
+}
+
```

Comment:
To keep some order in code, would you mind to put `registerDependencies` above `restartDependencies`? Kinda seems not in order the way it is now

Reply:
done